packages/@aws-cdk/core/README.md

```diff
@@ -1302,4 +1302,109 @@ permissions boundary attached.
 
 For more details see the [Permissions Boundary](https://docs.aws.amazon.com/cdk/api/v2/docs/aws-cdk-lib.aws_iam-readme.html#permissions-boundaries) section in the IAM guide.
 
+## Policy Validation
+
+If you or your organization use any policy validation tool, such as
+[CloudFormation
+Guard](https://docs.aws.amazon.com/cfn-guard/latest/ug/what-is-guard.html) or
+[OPA](https://www.openpolicyagent.org/), to define constraints on your
+CloudFormation template, you can integrate them with the CDK at synthesis time.
+By using the appropriate plugin, you can make the CDK application check the
+generated CloudFormation template against your policies immediately after
+synthesis. If there are any violations, the synthesis will fail and a report
+will be printed to the console.
+
+> **Note**
+> This feature is considered experimental, and both the plugin API and the
```

Comment:
From my experience we've done a little of both (and i've received feedback in both directions lol). I'm not sure there is a big difference between doing `myApiBeta1 => myApi` vs `myApi = myApiV2` tbh

Reply:
Is the plan to launch with `policyValidation` and then maybe change to `poilicyValidationV2`? If that is the case, this note is misleading because it suggests the existing API can break. If the plan is to possibly break existing API, I would advice against that.

In any case, I think that `policyValidationBeta1 => policyValidation` is better than `policyValidation => policyValidationV2`:

- It makes it clear that the current API is a beta version.
- It allows us to get feedback before we use up the "pretty" name, that we actually want.